core/baseline-core-v1.0-psd01.md

```diff
@@ -2293,12 +2285,18 @@ A workstep MUST have an input, one or more process steps, and an output.
 
 *This is just a well-known convention from business process management frameworks.*
 
+[[R211]](#r211) Testability: A valid workstep is a function that transforms a given input, using one or more transformations of the input into an output. An invalid workstep either does not have an input, or transformations of the input into an output, or an output, or any combination of the three components.
+
 #### **[R212]** 
 The input of a workstep MUST represent a new, proposed state of a state object compliant with the agreement between the agreement counterparties.
```

Comment:
Seems like this is conflating at least two things: 
- The output must represent a new valid state object different from the input (the fact that it must be different repeats the transformation requirement in R211)
- The output must conform to the agreement (which is also what R213 says)

Reply:
I think the new wording you suggested is fine.